mountpoint-s3/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 * Add support for appending to objects originally uploaded with a CRC64-NVME checksum. ([#1235](https://github.com/awslabs/mountpoint-s3/pull/1235))
 
+### Other Changes
+* Fix an issue where file names were not expelled from negative cache.
```

Comment:
We should include a link to this PR, like in the other entries:
```suggestion
* Fix an issue where file names were not expelled from negative cache. ([#1250](https://github.com/awslabs/mountpoint-s3/pull/1250))
```

Reply:
Also:
* why "file names"? Doesn't it apply to directories as well?
* we should qualify when the issue occurs.